.gitignore

```diff
@@ -63,7 +63,8 @@ build-iPhoneSimulator/
 .idea/
 .vscode/
 # node_modules
-/atd-vze/node_modules
+/viewer/node_modules
+/editor/node_modules
```

Comment:
added this line here for the editor. felt like it belonged here alongside the other react app.